moped-editor/src/views/projects/projectView/ProjectTeam/ProjectTeamRoleMultiselect.js

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+
+import {
+  Checkbox,
+  FormControl,
+  FormHelperText,
+  Input,
+  ListItemText,
+  MenuItem,
+  Select,
+  Typography,
+} from "@mui/material";
+import makeStyles from '@mui/styles/makeStyles';
+import { useGridApiContext } from "@mui/x-data-grid-pro";
+
+
+const useStyles = makeStyles((theme) => ({
+  formControl: {
+    margin: theme.spacing(1),
+  },
+  noLabel: {
+    marginTop: theme.spacing(3),
+  },
+  infoIcon: {
+    color: theme.palette.action.disabled,
+  },
+}));
+
+const ProjectTeamRoleMultiselect = ({ id, field, roles, value }) => {
+  const rolesArray = value.map((role) => role.project_role_id);
```

Comment:
it's a tiny performance drag, but since this mapping is firing on every keystroke into the form inputs, it might be good to memoize it 🤷